js/mainViewModel.js

```diff
@@ -1,108 +1,54 @@
+var renderDom = require('noddity-render-dom')
+var routing = require('./routing')
 var Ractive = require('ractive')
-var config = noddityConfig
-var Renderer = require('noddity-renderer')
+var config = window.noddityConfig
+Ractive.DEBUG = config.debug
 
-function noop() {}
-
-module.exports = function MainViewModel(butler, linkifyEmitter, routingEmitter) {
-	var renderer = new Renderer(butler, linkifyEmitter.linkify)
-	var changePostInRactive = null
+module.exports = function MainViewModel(butler, linkifyEmitter) {
+	var options = {
+		butler: butler,
+		linkifier: linkifyEmitter,
+		el: 'body',
+		data: config
+	}
 
 	var titleRactive = new Ractive({
 		el: 'title',
-		template: '{{title}}{{#page}} | {{page}}{{/page}}',
-		data: {
-			title: config.title
-		}
+		template: document.title
 	})
 
-	var mainRactive = new Ractive({
-		el: 'body',
-		data: Object.create(config)
+	linkifyEmitter.on('link', function(pageName) {
+		butler.getPost(pageName, function () {})
 	})
 
-	butler.getPost(config.template, function(err, post) {
+	renderDom('post', options, function (err, setCurrent) {
 		if (err) {
-			doSomethingAboutThisError(err)
-		} else {
-			mainRactive.resetTemplate(post.content)
+			console.error(err)
+			document.body.innerHTML = '<h1>ERROR</h1>' + err.message
 		}
-	})
+		var routingEmitter = routing(butler.getPost)
 
-	function doSomethingAboutThisError(err) {
-		console.log(err)
-	}
+		routingEmitter.on('current', function (postTitle) {
```

Comment:
It's a bit fiddly, but elsewhere in Noddity the "title" is the pretty string in the post metadata, while "name" is the filename of the post - this should be renamed for clarity